From f8b8ec13f5c740be496ee16a4c252eee30d07b4c Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sun, 5 Apr 2026 02:26:16 +0800
Subject: [PATCH] AOSCOS: Makefile: do not hard-code -fstack-protector-strong

On architectures like IA-64, there is no support for such a flag and
Autobuild passes it conditionally anyway.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 43b128e5..85a2f7a9 100644
--- a/Makefile
+++ b/Makefile
@@ -40,7 +40,7 @@ ifeq ($(origin CC),default)
 CC := $(CROSS_COMPILE)gcc
 endif
 CXFLAGS ?= -ggdb
-CWFLAGS ?= -Wall -Werror -Wstrict-prototypes -Wextra -Wno-unused-parameter -Wformat -Wformat-security -Werror=format-security -fstack-protector-strong -fPIE -Warray-bounds
+CWFLAGS ?= -Wall -Werror -Wstrict-prototypes -Wextra -Wno-unused-parameter -Wformat -Wformat-security -Werror=format-security -fPIE -Warray-bounds
 ifdef WARN_UNUSED
 CWFLAGS += -Wp -O3
 endif
-- 
2.55.0

